src/Rule/ContainsNumericCharacters.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Morebec\Validator\Rule;
+
+
+use Morebec\Validator\ValidationRuleInterface;
+
+class ContainsNumericCharacters implements ValidationRuleInterface
+{
+    /**
+     * @var int
+     */
+    private $numberCharacters;
+    /**
+     * @var bool
+     */
+    private $strict;
+    /**
+     * @var string|null
+     */
+    private $message;
+
+    /**
+     * ContainsNumericCharacters constructor.
+     * @param int|null $numberCharacters
+     * @param bool|null $strict
+     * @param string|null $message
+     */
+    public function __construct(
+        ?int $numberCharacters = 1,
```

Comment:
Can you ensure that the value of the parameter is always a positive integer or zero?